### PR TITLE
PREOPS-854: Contribute to early science document 

### PR DESCRIPTION
**Pull request by ebellm**

### CODE REVIEW

File: RTN-011.tex [hunk 1]
```diff
@@ -47,9 +47,13 @@
...
-
+Prior to \drone, scientific investigations may be undertaken with data processed in Alert Production and Solar System Processing as well as \dpvs of the commissioning data.
+Alerts of transient, variable, and/or moving objects are the only data product that will be immediately available (within 60 seconds of image readout) and publicly shareable, i.e not subject to a proprietary period. 
+Alert Production requires high-quality template images of the sky, and thus full volume and full fidelity alerts will not be available until after \drone.
+However, \ro is planning a process of incremental template generation to maximize the time-domain and solar system science achievable in the first year of operations.
+The worldwide community is actively preparing to process the LSST alert stream and use it to generate groundbreaking scientific results. 
+This note describes the \ro plan for ensuring \es. 
+It is a living document that will evolve over the course of the remainder of the construction project and up until \drone. 
 }
```
Comment (leannep) [inline RTN-011.tex:58]: Reads well. In another ticket, I've updated the abstract to be less detailed and added a summary section. I will work your version into this.

File: alertproduction.tex [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
+
+Both Alert Production and Solar System Processing thus depend on the presence of template images.
+During steady-state operations, these templates will be constructed during the annual Data Releases and will built from the best available subset of images taken.
+The input images for DRP-produced templates will accordingly have very good seeing and  comprehensive spatial coverage.
+Coadding multiple images enables artifact rejection  \citedsp{DMTN-080} and reduces noise.
+All of these template characteristics all help to ensure that image differencing is highly complete and highly pure.
+
+To enable alert production to proceed during commissioning and early operations, it is necessary to accept templates of lower quality.
+Because we have a smaller set of input images to choose from and uncertain knowledge about future observations, on-the-fly (or incremental) template generation necessarily must balance the trade off of earlier template availibility against template quality and spatial completeness.
```
Comment (leannep) [inline alertproduction.tex:22]: I wonder of we should use this document to drop "incremental template generation". My original idea behind the term was to incrementally build a template library duuring the course on year one. This is not how it is understood though and I think leads to confusion.

File: alertproduction.tex [hunk 3]
```diff
@@ -1,11 +1,39 @@
...
+
+To enable alert production to proceed during commissioning and early operations, it is necessary to accept templates of lower quality.
+Because we have a smaller set of input images to choose from and uncertain knowledge about future observations, on-the-fly (or incremental) template generation necessarily must balance the trade off of earlier template availibility against template quality and spatial completeness. 
+A template constructed today will enable alerts tomorrow, but that template might produce fewer or lower-quality alerts than one constructed from more data in a week or a month's time.
+Substantial validation will be required to determine when to build incremental templates to maximize the net throughput of \es.
+Nevertheless our goal is to enable Alert Generation to begin as soon as the data are scientifically useful.
+
+Coadding multiple images is formally required due to the noise-level requirements placed on the DM system.
+Additionally, the LSST survey is heavily dithered, so without coadding many images onto a common sky plane it both difficult and inefficient to obtain image differences for a new pointing from past single images.
```
Comment (leannep) [inline alertproduction.tex:28]: it is both difficult .....

File: commissioning.tex [hunk 4]
```diff
@@ -8,11 +8,19 @@ \subsection{Schedule}
...
+
+\subsection{Alert generation during commissioning}
+
+Due to the need to verify the instrument characteristics, template quality, and image differencing and Real/Bogus performance, real-time alerts will not be immediately available during the commissioning period. 
+Where the accumulated ComCam data is sufficient for alert generation, we expect to provide alerts at high latency (weeks--months). 
+The goal for these ``canned'' alerts is to enable alert brokers and science users to understand their characteristics and to help to validate their quality rather than to enable rapid followup and \es per se.
+During LSSTCam commissioning we intend to generate incremental templates over the maximal sky area supported by the available observations.
```
Comment (leannep) [inline commissioning.tex:25]: incrementally generate templates over ....

**Review by leannep — CHANGES_REQUESTED** [review]
Once you have merged, I'll have to manually do soome merging of your updates with mine. I will get you to review.

File: alertproduction.tex (hunk 2 above)
Comment (ebellm) [reply alertproduction.tex:22]: Yes, let's discuss this further--I'm not immediately sure of a better candidate but I can see how it is confusing.

**Pull request merged**